test/runnable/mangle.d

```diff
@@ -618,6 +618,12 @@ static assert(funcd.mangleof == "_D6mangle5funcdFPFZNnZi");
 
 /***************************************************/
 
+struct S21753 { void function() f1; }
+void fun21753(S21753 v)() {}
+alias fl21753 = (){};
+static assert((fun21753!(S21753(fl21753))).mangleof == "_D6mangle__T8fun21753VSQv6S21753S1_DQBi10" ~ fl21753.stringof ~ "MFNaNbNiNfZvZQCaQp");
```

Comment:
@BorisCarvajal: With LDC, I'm somehow getting
```
actual:   _D6mangle__T8fun21753VSQv6S21753S1f_DQBj10__lambda71MFZvZQBtFNaNbNiNfZv
expected: _D6mangle__T8fun21753VSQv6S21753S1f_DQBj10__lambda71MFNaNbNiNfZvZQCbQp
```

Any ideas?

Reply:
The lambda on LDC has no attributes ('FZv' instead of 'FNaNbNiNfZv'). Also it misses a backreference because of that.

Reply:
@kinke, I found this [line](https://github.com/ldc-developers/ldc/blob/aeeb90b241736bfe3706cf56b2955c40bb3d8ba8/dmd/expression.d#L4057) to be the cause, it's specific to LDC so I don't know what it really does.

Reply:
Thx so much!